tools/seadsa.cc

```diff
@@ -2,70 +2,66 @@
 // seadsda -- Print heap graph computed by DSA
 ///
 
-#include "llvm/LinkAllPasses.h"
-#include "llvm/IR/LegacyPassManager.h"
 #include "llvm/IR/LLVMContext.h"
+#include "llvm/IR/LegacyPassManager.h"
```

Comment:
I assume nothing changed in this file and this is just formatting. Let me know if that is not the case.